packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/legacy/index.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+import 'angular-translate';
+
+import component from './component';
+import './styles.less';
+
+const moduleName = 'ovhManagerDedicatedCloudLegacy';
```

Comment:
The module name must reflect the architecture.

```suggestion
const moduleName = 'ovhManagerDedicatedCloudDatacenterBackupLegacy';
```